client/src/components/BackupCodes/Register.js

```diff
@@ -114,13 +114,15 @@ class Register extends Component {
             'somewhere safe, as they will not be viewable after leaving this page.'
         )}
         &nbsp;
-        <a
-          href={method.supportLink}
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          {i18n._t('MFARegister.HELP', 'Find out more')}
-        </a>
+        {supportLink &&
+          <a
+            href={supportLink}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            {i18n._t('MFARegister.RECOVERY_HELP', 'Learn more about recovery codes.')}
```

Comment:
Do you want to use `method.supportText` here? I see you've added it to the PHP class